uint8 TYPE_N_A=0
uint8 TYPE_TIMEOUT=1
uint8 TYPE_WARNING=2
uint8 TYPE_OK=3
uint8 type
string title
float32 hz
